### PR TITLE
Fix 3551 –  Warnings when adding new migration file

### PR DESCRIPTION
**Pull request by bgeuken**

### CODE REVIEW

File: src/api/app/models/package.rb [hunk 1]
```diff
@@ -42,6 +42,13 @@ class ReadSourceAccessError < APIException
   class IllegalFileName < APIException; setup 'invalid_file_name_error'; end
   class PutFileNoPermission < APIException; setup 403; end
 
+  unless defined?(BINARY_EXTENSIONS)
+    BINARY_EXTENSIONS = %w{.0 .bin .bin_mid .bz .bz2 .ccf .cert .chk .der .dll .exe .fw
```
Comment (ChrisBr) [inline src/api/app/models/package.rb]: Have you tried if 
```BINARY_EXTENSIONS ||= ...```
would also solve the issue? Would be a lot nicer in my opinion :smile:

Comment (bgeuken) [inline src/api/app/models/package.rb]: That would also work. I just have a slight preference for ``` defined?(BINARY_EXTENSIONS) ``` in this case^^

**Comment by bgeuken** [comment]
@ChrisBr Updated. Please review again.

File: src/api/app/models/package.rb (hunk 1 above)
Comment (ChrisBr) [inline src/api/app/models/package.rb]: And why or rather why did you change it nevertheless :+1:
